HisashiburiDana.Application/Validators/ResetPasswordRequestValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+using HisashiburiDana.Contract.Authentication;
+using HisashiburiDana.Contract.Users;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HisashiburiDana.Application.Validators
+{
+    public class ResetPasswordRequestValidator : AbstractValidator<ResetPasswordRequest>
+    {
+        public ResetPasswordRequestValidator()
+        {
+            RuleFor(x => x.OldPassword).NotNull().NotEmpty();
+            RuleFor(x => x.NewPassword).NotEmpty().NotNull();
+            RuleFor(x => x.ConfirmPassword).NotEmpty().NotNull();
```

Comment:
To make the code neater. You can put a validator here that new password and confirm password must be the same. You can also put that old password must not be equal to new password. Instead of putting them in the application layer, after hashing